best_practices/forms.rst

```diff
@@ -42,14 +42,9 @@ form in its own PHP class::
                 'data_class' => 'AppBundle\Entity\Post'
             ));
         }
-
-        public function getName()
-        {
-            return 'post';
-        }
     }
 
-To use the class, use ``createForm`` and instantiate the new class::
+To use the class, pass the class name to the ``createForm()`` method::
```

Comment:
I think the country you live in isn't important here, but I agree that it's a difficult acronym for people not well known to PHP 5.3 namespaces and such (and that still is quite a big group of devs).